src/proto/net.proto

```diff
@@ -2,7 +2,11 @@ syntax = "proto2";
 
 package kuuhaku.proto;
```

Comment:
Since we're not really using `kuuhaku` namespace, maybe just do `package proto`?